dockerenforcer/killer.py

```diff
@@ -38,7 +40,9 @@ def register_killed(self, container, reason):
             if container.cid in self.__killed_containers.keys():
                 self.__killed_containers[container.cid].record_new(reason)
             else:
-                self.__killed_containers[container.cid] = Stat(container.params['Name'], reason)
+                self.__killed_containers[container.cid] = Stat(
+                    container.params['Name'], reason, container.params['Config']['Image'], container.params['Config']['Labels']
```

Comment:
this can't be left like that: case above uses `record_new` method, but this one creates in place. `record_new` must handle additional parameters too